### DISCUSSION TITLE
Displaying abandoned mines on mini-map

### DISCUSSION THREAD

**Discussion by Districh-ru** [ihhub/fheroes2#6807] [Ideas]
Currently in fheroes2 the abandoned mines and Haunted mines are not shown on mini-map.
In original game the initially abandoned mines are also not shown on mini-map. But mines abandoned with the Haunt spell are shown on mini-map just like any other non-captured mine.
Should we display them on mini-map in fheroes2 or not?

**Comment by LeHerosInconnu** [comment]
Hello @Districh-ru,

> Currently in fheroes2 the abandoned mines are not shown on mini-map. In original game the initially abandoned mines are also not shown on mini-map. But mines abandoned with the Haunt spell are shown on mini-map just like any other non-captured mine. Should we display them on mini-map in fheroes2 or not?

No, abandoned mines should not be displayed on the minimap.
They should be considered as bonuses to be discovered through exploration. :)

Edit:
Haunted mines shoud be displayed as neutral mines.
Once restored, abandoned mines should be displayed on the minimap.

Edit 2:
Related issue: https://github.com/ihhub/fheroes2/issues/6808.

Edit 3:
Related issue: https://github.com/ihhub/fheroes2/issues/6810.

**Reply by Districh-ru** [reply]
Hello @LeHerosInconnu.
Thanks for the quick reply!
Lets implement the original game behavior. :)

**Reply by zenseii** [reply]
A bit off-topic but how are these mines handled on the view world map then? Like when using the View Mines spell.

**Reply by Districh-ru** [reply]
Hello, @zenseii!
Thanks for asking the proper question!
In original the initially abandoned mine  is not show in the View World. In fheroes2 it is the same.
But Haunted mines are show in the original game like any other non-captured mine and in fheroes2 it is not shown at all.
I'll fix it in the PR with the fix for proper display of haunted mines on mini-map.

**Reply by zenseii** [reply]
Great! I figured there might be some things there too.
